BabelfishFeatures.cfg

```diff
@@ -611,6 +611,8 @@ supported-1.0.0=arg4=NO_SUPPORTED_ARGUMENTS_RIGHT_NOW
 
 [ALTER VIEW]
 rule=create_or_alter_view
+supported-4.6.0-4.*=*
+supported-5.2.0-5.*=*

```

Comment:
This line should be `supported-5.2.0=*` because it will be supported in every release from 5.2.0 onwards. By adding `-5.*`, you're saying that it will not be supported in 6.x but that is obviously wrong